logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/ClearConversationContent.kt

```diff
@@ -0,0 +1,83 @@
+package com.wire.kalium.logic.feature.conversation
+
+import com.benasher44.uuid.uuid4
+import com.wire.kalium.logic.CoreFailure
+import com.wire.kalium.logic.data.asset.AssetRepository
+import com.wire.kalium.logic.data.client.ClientRepository
+import com.wire.kalium.logic.data.conversation.ConversationRepository
+import com.wire.kalium.logic.data.id.ConversationId
+import com.wire.kalium.logic.data.message.Message
+import com.wire.kalium.logic.data.message.MessageContent
+import com.wire.kalium.logic.data.user.AssetId
+import com.wire.kalium.logic.data.user.UserRepository
+import com.wire.kalium.logic.feature.message.MessageSender
+import com.wire.kalium.logic.functional.Either
+import com.wire.kalium.logic.functional.flatMap
+import com.wire.kalium.logic.functional.fold
+import kotlinx.datetime.Clock
+
+internal class ClearConversationContent(
+    private val conversationRepository: ConversationRepository,
+    private val assetRepository: AssetRepository
+) {
```

Comment:
Instead of having this internal UseCase, can't we instead rename it to `ClearLocalConversationContentUseCase`?

This way it can be re-used externally too and apps can choose whether to clear only locally or on all devices by using one usecase or the other.

Reply:
I think I am going to leave it as it for now, I have abit trouble finding the right name ClearLocalConversationContentUseCase does sound right for me but a use case broadcasting it to other clients ? How about naming that one ? ClearConversationContentAcrossDevicesUseCase ?

Reply:
if you got something fitting, let me know :)